lib/rust/ensogl/component/toggle-button/src/lib.rs

```diff
@@ -66,6 +66,7 @@ ensogl_core::define_endpoints! {
         mouse_out  (),
         is_hovered (bool),
         is_pressed (bool),
+        clicked    (),
```

Comment:
Is it really necessary? I think `is_pressed.on_true()` has the same behavior.

Reply:
Good catch!